src/API/src/db/shared/entities/species.entity.ts

```diff
@@ -42,4 +43,15 @@ export class Species extends BaseEntity {
     nullable: false,
   })
   reference: Reference;
+
+  @OneToMany(
+    () => RecordedSpecies,
+    (recorded_species) => recorded_species.species,
+    {
+      eager: true,
+      cascade: true,
+      nullable: false,
+    },
+  )
+  recordedSpecies: RecordedSpecies[];
```

Comment:
I don't think we need this here - does it work without the recordedSpecies on the species entity?